tests/redis/test_manager.py

```diff
@@ -84,6 +85,21 @@ async def test_cache_manager_save_and_get_many_objects_with_keys_success(cache_m
     assert response == test_dtos
 
 
+async def test_cache_manager_save_and_get_many_objects_with_keys_as_it_is_success(cache_manager: CacheManager) -> None:
```

Comment:
Instead of having a separate test, you can parametrise the existing ones to make sure all behaviours are cross-checked.